.github/workflows/publish_plugin_to_hub_duckdb.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     outputs:
       plugin_name: ${{ steps.split.outputs.plugin_name }}
       plugin_kind: ${{ steps.split.outputs.plugin_kind }}
+      plugin_kind_plural: ${{ steps.split.outputs.plugin_kind_plural }}
```

Comment:
What do we need the plural form for?

Reply:
The Website directories are in the format `docs/plugins/sources` and `docs/plugins/destinations`, and we copy the docs from them.
I could hard code the values too since this workflow is scoped to a specific plugin, but that actually makes it harder to sync with https://github.com/cloudquery/cloudquery/blob/49ab5669d0bef1ab3ce907238feb2b9664a4b802/.github/workflows/publish_plugin_to_hub.yml